config.go

```diff
@@ -196,6 +196,8 @@ type Config struct {
 		// (defaults to hashing the message key). Similar to the `partitioner.class`
 		// setting for the JVM producer.
 		Partitioner PartitionerConstructor
+		// Controls whether the partitioner is rack-aware. This also affects custom partitioners.
+		PartitionerRackAware bool
```

Comment:
🤔 Do we have any Java-side Kafka config value to inform this config name? Does Java’s Kafka even support this?